precise-revoke-redis-sessions/redis.lua

```diff
@@ -0,0 +1,24 @@
+local cursor = 0
+local dels = 0
+local user = ARGV[1]
+
+if string.len(user) < 5 then
+  return "user too short"
+end
+
+repeat
+    local result = redis.call('SCAN', cursor, 'MATCH', 'sess:*', 'COUNT', 1000)
+
+    for _, key in ipairs(result[2]) do
+        local token = redis.call('GET', key)
+        if string.find(token, user) then
```

Comment:
Would it make sense to just wrap the value between quotes?

```suggestion
        if string.find(token, '"' .. user .. '"') then
```

This way, we can ensure to have matched the exact value, wrapped with quotes, and avoid bad match.